<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>new_session_window</class>
 <widget class="QWidget" name="new_session_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Новый сеанс</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Сохранение</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Путь</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="path_line_edit"/>
          </item>
          <item>
           <widget class="QToolButton" name="path_tool_button">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Имя файла</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="filename_line_edit"/>
          </item>
          <item>
           <widget class="QPushButton" name="generate_name_button">
            <property name="text">
             <string>generate</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Подключенные датчики</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QCheckBox" name="s1_checkbox">
          <property name="text">
           <string>№1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="s2_checkbox">
          <property name="text">
           <string>№2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="s3_checkbox">
          <property name="text">
           <string>№3</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="s4_checkbox">
          <property name="text">
           <string>№4</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="s5_checkbox">
          <property name="text">
           <string>№5</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="updtade_sensors_button">
          <property name="text">
           <string>Обновить</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Описание сессии</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QTextBrowser" name="session_comment_editor">
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="accept_button">
       <property name="text">
        <string>Принять</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_button">
       <property name="text">
        <string>Отменить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
